src/window.jl

```diff
@@ -106,6 +116,15 @@ body(win::Window, html::String) =
 const initcss = """
   <style>html,body{margin:0;padding:0;border:0;text-align:center;}</style>
   """
+body(win::Window, x) =
+  # FIXME: use tohtml
```

Comment:
`body` is only really meant as a very low-level method, the caller should figure out the best way to represent something by itself I think. `tohtml` is only really an implementation detail of BlinkDisplay and isn't meant to be general; other displays probably have their own versions of it.